tests/sles4sap/hana_install.pm

```diff
@@ -258,8 +264,9 @@ sub run {
     my @hdblcm_args = qw(--autostart=n --shell=/bin/sh --workergroup=default --system_usage=custom --batch
       --hostname=$(hostname) --db_mode=multiple_containers --db_isolation=low --restrict_max_mem=n
       --userid=1001 --groupid=79 --use_master_password=n --skip_hostagent_calls=n --system_usage=production
-      --components=server);
+    );
     push @hdblcm_args,
+      "--components=server,client",
```

Comment:
The `client` part is needed by this other test module https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/monitoring_services.pm#L52

I propose to add a comment in the code to explain it

Reply:
> The `client` part is needed by this other test module https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/sles4sap/monitoring_services.pm#L52
> 
> I propose to add a comment in the code to explain it

Done.